docs/integrations.md

```diff
@@ -88,6 +85,11 @@ jobs:
       #  if: always()
 ```
 
+#### Inline PR Comments (beta)
+Set `GITHUB_TOKEN` (commented out in the code above) to leave inline comments on your pull requests.
+Comments are left when Semgrep CI finds a result that blocks CI.
```

Comment:
This surfaces a good product question! Should inline comments only be triggered by "block CI" results? I'd love to chat this week, I've gotten feature requests for notify-only WITH inline pr comments :)

Reply:
That is a very good question - I wonder which is more intuitive for the user. I think the reasoning for implementing it firing on "Block CI" is that an inline PR comment does "block" a PR.

Also, what exactly is "block CI" for in general? Findings that should cause CI to fail? (In which case, are rules/rulesets specified to not block CI effectively disabled?)

Reply:
Does an inline PR comment have to block a PR or can it be informational (i.e. done outside of a PR-review)? Rules / rulesets specified not to block CI are effectively informational.

Reply:
Just looking around, it seems you can "Add single comment", which doesn't require a review. The comments right now do start a review (which is not something I remember setting, so it must just be the endpoint we're using), but I'm sure there exists an API endpoint that allows you to just leave an inline comment.